lib/hexpm_web/controllers/api/docs_controller.ex

```diff
@@ -34,27 +34,14 @@ defmodule HexpmWeb.API.DocsController do
     request_id = List.first(get_req_header(conn, "x-request-id"))
 
     log_tarball(organization.name, package.name, release.version, request_id, body)
+    Hexpm.Repository.Releases.publish_docs(package, release, body, audit: audit_data(conn))
 
-    case HexpmWeb.DocsTar.parse(body) do
```

Comment:
@ericmj fyi DocsTar was now unused and I removed it on master.

Reply:
Thanks!